### DISCUSSION TITLE
想参考archery搞个巡检工具

### DISCUSSION THREAD

**Discussion by unknowissue** [hhyo/Archery#1564] [Q&A]
平时要巡检Oracle数据库实例好多，我想参考archery搞个巡检工具，这样开发快一点。
介意么？

**Comment by hhyo** [comment]
这个按照开源协议来就行，不过Archery的前端比较陈旧，可能会导致开发效率低，建议可以尝试下新的前端框架啥的😂

**Reply by unknowissue** [reply]
嗷嗷，主要是我之前没干过开发，archery可能熟悉一点，能看着改一改，也就是我改脱敏那段的水平😂。。。
有什么推荐的前端框架么，我试一试。
😀

**Reply by LeoQuote** [reply]
前端现在是群魔乱舞啊, 没什么特别要求比较建议 nuxt 或者 next + elementui 或者 vuetify 

nuxt 就是目录结构什么的清晰一些, elementui 国内用的多, vuetify 是 material design, 看需求和哪款你觉得好看

**Reply by unknowissue** [reply]
好滴，我看看，阿里嘎多

**Comment by unknowissue** [comment]
哈哈哈哈哈，我看了看，这些前端我好像驾驭不了
我先魔改一版吧，先满足需求先，手动执行脚本和生成报告太难了。
